### PR TITLE
refactor(behavior_velocity_planner): rework parameters

### PR DESCRIPTION
**Pull request by yuntianyi-chen**
## Description

Refactor the ROS Node configuration layout for the behavior_velocity_planner package according to the [guideline](https://github.com/orgs/autowarefoundation/discussions/3433).
- Create the schema file.
- Refactor the parameter file.
- Remove the default values from the source code to avoid duplication.

## Related links

None

## Tests performed

Output an error when launching this node individually since this launch file is dependent on other nodes.

## Notes for reviewers

None

## Interface changes

- Create one configuration file `planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json`
- Parameter values are now required to be passed to the node when launched.

## Effects on system behavior

None

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].
- [x] The PR has been properly tested.
- [x] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by satoshi-ota** [comment]
@yuntianyi-chen Thanks for your contribution. Unfortunatelly, it seems that this PR has conflicts. Could you fix it at first? After that I'll review this asap.

**Comment by yuntianyi-chen** [comment]
> @yuntianyi-chen Thanks for your contribution. Unfortunatelly, it seems that this PR has conflicts. Could you fix it at first? After that I'll review this asap.

I've resolved the conflicts. Please review them!

**Review by satoshi-ota — APPROVED** [review]
LGTM :+1:

File: planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          "default": "5.0",
+          "description": "extend length of stop line"
+        },
+        "max_jerk": {
+          "type": "number",
+          "default": "-5.0",
+          "description": "max jerk of the vehicle"
+        },
+        "is_publish_debug_path": {
+          "type": "number",
```
Comment (xmfcx) [inline planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json:45]: @yuntianyi-chen could you fix this?

It fails this check: https://github.com/autowarefoundation/autoware.universe/actions/runs/6975071500/job/18981661740?pr=5263#step:3:47

Comment (yuntianyi-chen) [reply planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json:45]: I started a new PR and it's already modified and merged in the main branch.

Comment (xmfcx) [reply planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json:45]: You are right, I'm sorry I didn't realize it. <3

**Pull request merged**